src/e2e-test/resources/pluginParameters.properties

```diff
@@ -335,3 +335,11 @@ bqExecuteCountDMLUpsertInsert=SELECT COUNT(*) FROM `PROJECT_NAME.DATASET.TABLENA
 bqExecuteCountDMLUpsertUpdate=SELECT COUNT(*) FROM `PROJECT_NAME.DATASET.TABLENAME` WHERE Id=101 AND Value=5000 AND UID='UPDATED RECORD'
 bqExecuteInsertFile=testdata/BQExecute/BQExecuteInsertFile
 ## BQEXECUTE-PLUGIN-PROPERTIES-END
+
+## DATASTORE-PLUGIN-PROPERTIES-START
+fieldName=firstName
+name=Antonio
+age=23
+result=
```

Comment:
Is the value blank?

Reply:
yes the value is blank for true or false. Blank value represents true.